src/content/reference/gsctl/create-keypair.md

```diff
@@ -29,13 +29,13 @@ $ gsctl create keypair --cluster "Cluster name"
 The output will look similar to the one below:
 
 ```nohighlight
-New key pair created with ID 114e2de892b2dcfdf96c92f3c69f39e564ecec1e and expiry of 30 days
+New key pair created with ID 114e2de892b2dcfdf96c92f3c69f39e564ecec1e and expiry of 1 day
```

Comment:
I think the ID in this output line should also be truncated, no? Unrelated maintenance.

Reply:
Yes, it does have to be truncated 👍